lib/checkPullRequestBranch.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const DISSALLOWED_BRANCH_PREFIXES = ['develop', 'release-', 'test-'];
+
+/**
+ * @param {import('probot').Context} context
+ */
+const checkBranch = async (context) => {
+  /**
+   * @type {import('probot').Octokit.PullsGetResponse} pullRequest
+   */
+  const pullRequest = context.payload.pull_request;
+  const branchName = pullRequest.head.ref;
+  const branchIsInvalid = DISSALLOWED_BRANCH_PREFIXES.some((prefix) => {
+    return branchName === prefix || branchName.startsWith(prefix);
```

Comment:
`branchName === prefix` - this isn't required, right? The second condition will work for both cases.

Reply:
True.
Removed.